lib/timed_semaphore.rb

```diff
@@ -8,17 +8,17 @@
 # @author ssuljic
 #
 # @example
-#   threads = []
-#   semaphore = TimedSemaphore.new(2, 3)
+# threads = []
```

Comment:
I'm following [yardoc](http://yardoc.org/) for documentation, and the [convention](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#example) for examples is to be indented 2 spaces. Could you please revert this?